packages/amplify_core/lib/src/config/auth/cognito/appsync.g.dart

```diff
@@ -21,7 +21,7 @@ Map<String, dynamic> _$CognitoAppSyncConfigToJson(
   final val = <String, dynamic>{
     'ApiUrl': instance.apiUrl,
     'Region': instance.region,
-    'AuthMode': _$APIAuthorizationTypeEnumMap[instance.authMode],
+    'AuthMode': _$APIAuthorizationTypeEnumMap[instance.authMode]!,
```

Comment:
Are we sure that authMode is always present? I'm wondering if the libraries have some sort of default in case it is missing in the config, in which case some customers may be successfully using a handwritten config.